src/jit/flowgraph.cpp

```diff
@@ -21804,7 +21804,7 @@ Compiler::fgWalkResult      Compiler::fgUpdateInlineReturnExpressionPlaceHolder(
             else
             {
                 // Just assign the inlinee to a variable to keep it simple.
-                tree->CopyFrom(comp->fgAssignStructInlineeToVar(tree, retClsHnd), comp);
+                tree = comp->fgAssignStructInlineeToVar(tree, retClsHnd);
```

Comment:
There must be a reason why we are reusing the same 'tree' instead of using the new gentree node returned by fgAssignStructInlineeToVar().  The name of this routine is fgUpdateInlineReturnExpressionPlaceHolder(), and the function header comment indicates that it is meant to replace in-line retexpr place holder node.

Besides, this change is now not required due to the way it is fixed in importer?